src/kvilib/ext/KviSharedFilesManager.cpp

```diff
@@ -65,15 +65,15 @@
 		If you have two file offers with the same name and different file, the remote user can
 		use an additional "size" parameter in the DCC GET request.[br]
 		[big]Security issues[/big]
-		This is a nice but unsecure method of sharing files.[br]
-		The user mask is a good protection but you have to use it properly!.[br]
+		This is a nice, but not a secure method of sharing files.[br]
```

Comment:
I dont like insecure, makes it sound like a insecure person with doubts about who they are... Not as Unsafe or not secure.